staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -109,7 +109,7 @@ func NewCmdCp(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobra.C
 			cmdutil.CheckErr(o.Run(args))
 		},
 	}
-	cmd.Flags().StringVarP(&o.Container, "container", "c", o.Container, "Container name. If omitted, prefer to use the kubectl.kubernetes.io/default-container annotation or the first container in the pod will be chosen")
+	cmd.Flags().StringVarP(&o.Container, "container", "c", o.Container, "Container name. If omitted, use the kubectl.kubernetes.io/default-container annotation for selecting the container to be attached or the first container in the pod will be chosen")
```

Comment:
@pacoxu @mengjiao-liu @eddiezane @soltysh 

I am wondering if would help creating a file to hold a const var for the message below as it must appear in attach.go, cp.go, exec.go, etc
 
"Container name. If omitted, use the kubectl.kubernetes.io/default-container annotation for selecting the container to be attached or the first container in the pod will be chosen"

Just thinking out loud, example:
staging/src/k8s.io/kubectl/pkg/cmd/util/default-container.go

This avoids duplicate code and help having a single place for maintenance.

Reply:
Agree.

It makes sense.

Reply:
I agree with you and I will change it. @dougsland